### PR TITLE
Change makefile and improve documentation

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,60 +1,69 @@
...
 CONSOLE=$(APP) bin/console
 
 .PHONY: clean
 
-all: start dependencies db perm
+help:           ## Show this help
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
 
-dependencies: composer yarn assets perm
+install:        ## [start deps db-create] Setup the project using Docker and docker-compose
```
Comment (HeahDude) [inline Makefile]: Duplicated task, please comment all the line

File: Makefile [hunk 2]
```diff
@@ -1,60 +1,69 @@
...
 	docker-compose down
-endif
 
-start:
-ifdef DOCKER
-	docker-compose up -d
-else
-	@bash -c "if [ -a vendor/autoload.php ]; then $(CONSOLE) server:run; fi;"
-endif
+deps:           ## [composer yarn assets-dev perm] Install the project PHP and JS dependencies
```
Comment (HeahDude) [inline Makefile]: same here

File: dev.dockerfile [hunk 3]
```diff
@@ -41,6 +41,7 @@ RUN apt-get update -q && \
         php7.1-mysql \
         php7.1-pdo \
         php7.1-phar \
+        php7.1-sqlite \
```
Comment (HeahDude) [inline dev.dockerfile]: tools?

File: docker-compose.override.yml.dist [hunk 4]
```diff
@@ -5,7 +5,7 @@ services:
         volumes:
             - "~/.composer:/root/.composer"
         ports:
-            - "80:80"
+            - "8000:80"
```
Comment (HeahDude) [inline docker-compose.override.yml.dist]: It now conflicts with the PMA below, change the other to 8081 then.

Comment (tgalopin) [inline docker-compose.override.yml.dist]: 8000 != 8080

Comment (HeahDude) [inline docker-compose.override.yml.dist]: True!

File: Makefile (hunk 1 above)
Comment (tgalopin) [inline Makefile]: Well, how would you do this and still have the comment?

File: dev.dockerfile (hunk 3 above)
Comment (tgalopin) [inline dev.dockerfile]: Nope, not for now.

**Pull request merged**